### PR TITLE
Newsweek Clone Bootstrap Project

### PR DESCRIPTION
**Pull request by Mohamed-js**
Project progress and milestones met:
1- Created the skeleton for the mobile site
2- Utilized Bootstrap for responsiveness
3- Added content and media to the site
4- Added some styles with CSS

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Mohamed-js  & @JohnHernandez-code**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] First of all, please fix all of your linters issues.

- [x] this project is called using bootstrap, and that means (as mentioned in the project requirements) that you have to use bootstrap classes for everything. that includes positioning, margins, paddings, layout, responsiveness, and whatever in between.
you are only allowed to use normal CSS classes when they are not already bootstrap classes.

- [x] the navbar should look exactly like the original one. 
the burger button only shows on the tablet and mobile views in the original page.
original: 
LISHDGLIUGLIDFUJHFOASLI

clone: 
asd


- [x] the original page has breakpoints at 992px for tablet and 768px for mobile. please make sure that your clone has the same breakpoints by using the bootstrap responsiveness prosperities.

- [x] the original page has no bootstrap cards. please make sure to remove those.
 


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by Mohamed-js** [comment]
Fiexed linters, BOOTSTRAP break points, the navbar and bootstrap elements.

File: index.html [hunk 1]
```diff
@@ -156,7 +156,7 @@ <h5>
               </div>
   
               <div class="card" style="width: 100%;">
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 2]
```diff
@@ -167,7 +167,7 @@ <h5 class="card-title">Card title</h5>
...
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 3]
```diff
@@ -25,7 +25,7 @@
         </form>
       </nav>
 
-      <nav class="navbar nav2 navbar-dark">
+      <nav class="navbar nav2 container navbar-dark">
         <form class="form-inline">
           <div>
             <a href="">Alexandria, EG</a>
```
Comment (umairarshadbutt) [inline index.html:31]: Add the description in `href` attribute

File: index.html [hunk 4]
```diff
@@ -40,7 +40,7 @@
         </form>
       </nav>
 
-      <nav class="nav3">
+      <nav class="nav3 container">
         <ul>
           <li><a href="">U.S.</a></li>
           <li><a href="">World</a></li>
```
Comment (umairarshadbutt) [inline index.html:46]: Add the description in `href` attribute

File: index.html [hunk 5]
```diff
@@ -466,7 +466,7 @@ <h5>
             <h6>THE DEBATE</h6>
             <div class="roow"> 
               <div>
-                <img src="imgs/main/12.webp" alt="" />
+                <img src="imgs/main/10.webp" alt="" />
```
Comment (umairarshadbutt) [inline index.html:517]: Add the description in `alt` attribute

File: index.html [hunk 6]
```diff
@@ -495,7 +495,7 @@ <h5>
 
             <div class="roow"> 
               <div>
-                <img src="imgs/main/12.webp" alt="" />
+                <img src="imgs/main/11.webp" alt="" />
```
Comment (umairarshadbutt) [inline index.html:546]: Add the description in `alt` attribute

File: index.html [hunk 7]
```diff
@@ -508,7 +508,7 @@ <h5>
             </div>
             <div class="roow"> 
               <div>
-                <img src="imgs/main/12.webp" alt="" />
+                <img src="imgs/main/13.webp" alt="" />
```
Comment (umairarshadbutt) [inline index.html:559]: Add the description in `alt` attribute

**Comment by Mohamed-js** [comment]
Removed cards and used only necessary CSS classes when they are not already bootstrap classes

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hello @Mohamed-js  and @JohnHernandez-code :wave: 
There are still some issues with your project that needs to be worked on 
- [ ] The body of your webpage should take up the total width of its screen. There are some margins to the left and right of the body of your clone.
- [ ] Your nav still doesn't look and behave like the original nav across all screen sizes, please make them look just like they do and behave in the original across different screen sizes.
- your clone 
Screenshot from 2020-12-09 18-48-15
- original
Screenshot from 2020-12-09 18-49-43
- [ ] You need to also work on the subscribe, newsletter, and footer section. Those section needs to look like they do in the original, especially how the image in the subscribe section looks like and the links and social icons in the footer. Also, remember they need to be responsive and look like they appear in the original website across different scene sizes.
- your clone 
Screenshot from 2020-12-09 18-55-28
- original
Screenshot from 2020-12-09 18-55-45
- [ ] In general, try to use the same font used in the original so that your webpage can appear as identical as possible to the original webpage.

Do request another review when you are done making these changes

slack(@ebeagusamuel)

**Pull request closed without merging**